server/src/schema.js

```diff
@@ -51,10 +70,15 @@ const resolvers = {
   Mutation: {
     createTask: async (obj, args, context, info) => {
       console.log("Create", args)
-      const result = await context.db('tasks').insert({ ...args, version: 1 }).returning('*').then((rows) => rows[0])
+      const result = await context.db('tasks').insert({ ...args,
+        version: 1
+      }).returning('*').then((rows) => rows[0])
       // TODO context helper for publishing subscriptions in SDK?
-      pubSub.publish(EVENTS.TASK.CREATED, {
-        taskCreated: result,
+      pubSub.publish(EVENTS.TASK.MUTATED, {
+        taskMutated: {
+          mutation: 'CREATED',
```

Comment:
kinda pollutes resolvers. Could we extract entire idea into separate file - subscriptions or something like that.